code.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+      "name": "plio",
+      "organization": "U.S. Geological Survey",
+      "description": "A planetary surface data input/output library written in Python.",
+      "version": "1.5.0",
+      "status": "Release",
+
+      "permissions": {
+        "usageType": "openSource",
+        "licenses": [
+          {
+            "name": "Public Domain, CC0-1.0",
+            "URL": "https://github.com/USGS-Astrogeology/plio/blob/master/LICENSE.md"
+          }
+        ]
+      },
+  
+      "homepageURL": "https://plio.readthedocs.io/en/latest/?badge=latest",
```

Comment:
I do not know if we can use this as the URL. Leave for now. I will have more information next week. @ladoramkershner FYI - we might have new documentation requirements coming down the pipe. Hence this comment. Nothing to do yet, but a heads up that things might be changing to ensure compliance.